server/graphql/v2/interface/Account.js

```diff
@@ -574,6 +575,15 @@ const accountFieldsDefinition = () => ({
       };
     },
   },
+  policies: {
+    type: new GraphQLList(Policy),
+    async resolve(account, _, req) {
+      if (req.remoteUser?.isAdminOfCollective(account)) {
+        return account.data?.policies || [];
+      }
+      return null;
```

Comment:
It won't be useful for now, but I think this field should be public. Everyone should be able to list the policies of the collective, as it's not private info and it might concern other people than collective admins in the future